apps/web-giddh/src/app/new-inventory/component/about-group-detail/about-group-detail.component.scss

```diff
@@ -19,6 +19,18 @@
                 }
             }
         }
+        .more-add-button{
```

Comment:
This css is already available in about-combo-details.component.scss, please don't create duplicate css, move this to common file.

Reply:
Updated